IPython/core/interactiveshell.py

```diff
@@ -803,7 +803,10 @@ def init_deprecation_warnings(self):
         This will allow deprecation warning of function used interactively to show
         warning to users, and still hide deprecation warning from libraries import.
         """
-        warnings.filterwarnings("default", category=DeprecationWarning, module=self.user_ns.get("__name__"))
+        #warnings.filterwarnings("default", category=DeprecationWarning, module=self.user_ns.get("__name__"))
+        if sys.version_info < 3.7:
```

Comment:
Change to:

`if sys.version_info < (3, 7)`

This is becasuse sys.version_info on py3 does not support the < operator (or that is my conclussion)

Reply:
Ok! Doing it right now. 

My biggest concern actually was if it resolved what the issue implied. How can I test my code?